src/components/menu-right/ContETFStaticEditor.vue

```diff
@@ -0,0 +1,191 @@
+import StuntSheet from "@/models/StuntSheet";
+
+/**
+ * View/Edit an ETF-Static continuity
+ */
+export default Vue.extend({
+  name: "ContETFStaticEditor",
+  props: {
+    continuityIndex: {
+      type: Number,
+      required: true,
+    },
+    dotTypeIndex: {
+      type: Number,
+      required: true,
+    },
+  },
+  computed: {
+    marchTypeOptions() {
+      return Object.values(MARCH_TYPES).map((marchType) => {
+        return {
+          friendlyName: `${marchType}`,
+          value: marchType,
+        };
+      });
+    },
+    directionOptions() {
+      return Object.values(DIRECTIONS)
+        .filter(Number.isInteger)
+        .map((dir) => ({
+          degrees: dir,
+          friendlyName: DIRECTIONS[dir as number],
+        }));
+    },
+    selectedMarchType: {
+      get(): MARCH_TYPES {
+        const continuity: ContETFStatic = this.$store.getters.getContinuity(
+          this.dotTypeIndex,
+          this.continuityIndex
+        );
+        return continuity.marchType;
+      },
+      set(marchType: MARCH_TYPES): void {
+        const continuity: ContETFStatic = this.$store.getters.getContinuity(
+          this.dotTypeIndex,
+          this.continuityIndex
+        );
+        continuity.marchType = marchType;
+        this.$store.commit("updateDotTypeContinuity", {
+          dotTypeIndex: this.dotTypeIndex,
+          continuityIndex: this.continuityIndex,
+          continuity,
+        });
+      },
+    },
+    selectedMarchingDirection: {
+      get(): number {
+        const continuity: ContETFStatic = this.$store.getters.getContinuity(
+          this.dotTypeIndex,
+          this.continuityIndex
+        );
+        return continuity.marchingDirection;
+      },
+      set(direction: number): void {
+        const continuity: ContETFStatic = this.$store.getters.getContinuity(
+          this.dotTypeIndex,
+          this.continuityIndex
+        );
+        continuity.marchingDirection = direction;
+        continuity.facingDirection = direction;
+        this.$store.commit("updateDotTypeContinuity", {
+          dotTypeIndex: this.dotTypeIndex,
+          continuityIndex: this.continuityIndex,
+          continuity: continuity,
+        });
+      },
+    },
+    selectedFacingDirection: {
+      get(): number {
+        const continuity: ContETFStatic = this.$store.getters.getContinuity(
+          this.dotTypeIndex,
+          this.continuityIndex
+        );
+        return continuity.facingDirection;
+      },
+      set(direction: number): void {
+        const continuity: ContETFStatic = this.$store.getters.getContinuity(
+          this.dotTypeIndex,
+          this.continuityIndex
+        );
+        continuity.facingDirection = direction;
+        this.$store.commit("updateDotTypeContinuity", {
+          dotTypeIndex: this.dotTypeIndex,
+          continuityIndex: this.continuityIndex,
+          continuity: continuity,
+        });
+      },
+    },
```

Comment:
Since we aren't using this yet, can you either delete out this code or comment it out?

Reply:
Done! Good catch.